src/sonic-yang-models/yang-models/sonic-tunnel.yang

```diff
@@ -0,0 +1,117 @@
+module sonic-tunnel {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-tunnel";
+    prefix tunnel;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+    
+    import sonic-types {
+        prefix stypes;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC DualToR tunnel data";
+
+    revision 2022-08-23 {
+        description
+            "Initial revision";
+    }
+
+    container sonic-tunnel {
+        container TUNNEL {
+            list TUNNEL_LIST {
+                max-elements 1;
+
+                key "mux_tunnel";
+                
+                leaf mux_tunnel {
+                    description "Name of MuxTunnel";
+                    type string {
+                        pattern "MuxTunnel[0-9]+";
+                    }
+                }
+
+                leaf dscp_mode {
+                    description "DSCP mode";
+                    type string {
+                        pattern "uniform|pipe";
+                    }
+                }
+
+                leaf src_ip {
+                    type inet:ipv4-address;
+                    description "optional source IPv4 address.";
+                }
+
+                leaf dst_ip {
```

Comment:
>dst_ip [](http://example.com/codeflow?start=21&length=6)

Is there any constrain on `dst_ip`? For example, must it be a neighbor IP?

Reply:
What would be the correct way to specify this? other than in the description

Reply:
To clarify, I am not asking you to modify yang model. The point of accurate yang model, is to capture exact backend (in this case orchagent/etc) behavior. If backend has the implementation to validate `dst_ip` with some constrain, yang model need to state the contain. If backend has no validation at all, current yang model is good enough.